### ISSUE TITLE
Override WR_DATABASE environment variable using database=

### ISSUE THREAD

**Issue by eric-valente** [awslabs/aws-data-wrangler#450]
I have an environment variable WR_DATABASE set to 'dev'.  

I was hoping I could override the environment variable by specifying the database like this code snippet, but it does not seem to be the case:

```
wr.s3.store_parquet_metadata(
    path ="s3://bucket",
    dataset=True,
    mode="overwrite",
    database="prod",  <--- override environment variable
    table=table
)
```

What is the cleanest/simplest way to change the database name on a one off basis and keep the environment variable?

**Comment by igorborgest** [comment]
Hi @melvinev, thanks for reaching out!

You should be able to do what you want. It is a bug and the commit above will fix it.

Do you mind to install it directly from our repository and check if is everything fine for your use case before the official release?
The idea is to publish it in the version `1.10.1` next Wednesday.

```sh
pip install git+https://github.com/awslabs/aws-data-wrangler.git --no-use-pep517
```

**Comment by igorborgest** [comment]
Released on version [1.10.1](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.10.1)
